frontend/src/pages/GettingStarted.test.tsx

```diff
@@ -46,181 +46,7 @@ describe('GettingStarted page', () => {
-                  class="link"
-                  href="#/pipelines"
-                >
-                  TFX pipeline demo with Estimator
-                </a>
-                 
-                <ul>
-                  <li>
-                    Classification pipeline with model analysis, based on a public BigQuery dataset of taxicab trips. 
-                    <a
-                      class="link"
-                      href="https://github.com/kubeflow/pipelines/tree/master/samples/core/parameterized_tfx_oss"
-                      rel="noopener"
-                      target="_blank"
-                    >
-                      source code
-                    </a>
-                  </li>
-                </ul>
-              </li>
-            </ul>
-            <br />
-            <p>
-              <strong>
-                Tutorials
-              </strong>
-               - Learn pipeline concepts by following a tutorial.
-            </p>
-            <ul>
-              <li>
-                <a
-                  class="link"
-                  href="#/pipelines"
-                >
-                  Data passing in python components
-                </a>
-                 
-                <ul>
-                  <li>
-                    Shows how to pass data between python components. 
-                    <a
-                      class="link"
-                      href="https://github.com/kubeflow/pipelines/tree/master/samples/tutorials/Data%20passing%20in%20python%20components"
-                      rel="noopener"
-                      target="_blank"
-                    >
-                      source code
-                    </a>
-                  </li>
-                </ul>
-              </li>
-              <li>
-                <a
-                  class="link"
-                  href="#/pipelines"
-                >
-                  DSL - Control structures
-                </a>
-                 
-                <ul>
-                  <li>
-                    Shows how to use conditional execution and exit handlers. 
-                    <a
-                      class="link"
-                      href="https://github.com/kubeflow/pipelines/tree/master/samples/tutorials/DSL%20-%20Control%20structures"
-                      rel="noopener"
-                      target="_blank"
-                    >
-                      source code
-                    </a>
-                  </li>
-                </ul>
-              </li>
-            </ul>
-            <p>
-              Want to learn more? 
-              <a
-                class="link"
-                href="https://www.kubeflow.org/docs/pipelines/tutorials/"
-                rel="noopener"
-                target="_blank"
-              >
-                Learn from sample and tutorial pipelines.
-              </a>
-            </p>
-          </div>
-        </div>
-      </div>
-    `);
+    expect(container).toMatchSnapshot();
```

Comment:
I shouldn't do refactoring and bug fix in one PR... now they are reverted

Reply:
I'm sorry, this wasn't my intention. 😢  There were merge conflicts, since my PR was modifying the sample, while you PR was removing it. I've tried to resolve the conflict, but it was not easy. I did not fully understand some of the snapshot-related changes you've made when removing the sample, so I've just reverted them to the initial state.

Let me know how I can help restore the refactoring.